pdfgen/Cargo.toml

```
@@ -11,7 +11,7 @@ license.workspace = true
 md5 = "0.7.0"
 hex = "0.4.3"
 image = "0.25.5"
-pdfgen_macros = {path = "../pdfgen_macros"}
+pdfgen_macros = { version = "0.1.0", path = "../pdfgen_macros" }
```

I think even better would be just using the workspace crate, like this:

```suggestion
pdfgen_macros = { workspace = true }
```